examples/persistence_transform.py

```diff
@@ -0,0 +1,41 @@
+"""
+========================================
+Persistence Transform in Mapcubes
+========================================
+
+Persistence Transform is a simple image processing technique that is useful for the visualization
+and depiction of gradually evolving structures.
+This example illustrates persistence transform using the AIA_193_CUTOUT series.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
+aiamapcube = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE,
+                           AIA_193_CUTOUT03_IMAGE, cube=True)
+
+###############################################################################
+# For a data set consisting of N images with intensity values I(x,y,t), the Persistence Map Pn is a
+# function of several arguments, namely intensity, location and time:
+# Pn(x,y,tn)=Q(I(x,y,t≤tn))
+persistence_maps = []
+persistence_maps.append(aiamapcube[0])    
```

Comment:
This is I think is redundant @nabobalis. Hence when I do the output, its basically the first map in the input that I am plotting.

Reply:
Ok but I don't understand what the PT is actually doing to the output. It doesn't look any better.